packages/flutter/lib/src/painting/border_radius.dart

```diff
@@ -26,6 +26,82 @@ abstract class BorderRadiusGeometry {
   /// const constructors so that they can be used in const expressions.
   const BorderRadiusGeometry();
 
+  /// Creates a [BorderRadius] where all radii are `radius`.
+  // The radius applies equally on all sides, so BorderRadiusDirectional is
+  // irrelevant in this case.
+  factory BorderRadiusGeometry.all(Radius radius) => BorderRadius.all(radius);
+
+  /// Creates a [BorderRadius] where all radii are [Radius.circular(radius)].
+  // The radius applies equally on all sides, so BorderRadiusDirectional is
+  // irrelevant in this case.
+  factory BorderRadiusGeometry.circular(double radius) => BorderRadius.circular(radius);
+
+  /// Creates a horizontally symmetrical border radius.
+  ///
+  /// Utilizing the `left` and `right` properties will return a [BorderRadius],
+  /// while `start` and `end` will yield a [BorderRadiusDirectional]. These
+  /// properties cannot be used interchangeably.
```

Comment:
I think it's good that you're calling out which subclasses will be created here in the docs, so that users are less confused that there are multiple ways to achieve the same thing :+1: